api/websocket/websocket.go

```diff
@@ -30,18 +31,21 @@ func NewServer(localNode *node.LocalNode, w vault.Wallet) *server.WsServer {
 }
 
 func SendBlock2WSclient(v interface{}) {
-	go func() {
-		PushSigChainBlockHash(v)
```

Comment:
Why not using n.GetSyncState()?

Reply:
You are right.
I pushed a new commit for it.